test/utils/columnStructure.js

```diff
@@ -0,0 +1,84 @@
+const columns = [
+  {
+    'Label': 'Order ID',
+    'Name': 'OrderID',
+    'Sortable': true,
+    'SortOrder': 1,
+    'SortDirection': 'Ascending',
+    'IsKey': true,
+    'Searchable': false,
+    'Visible': true,
```

Comment:
You don't need to add visible true, and many other properties. Check with @jpalcala this issue.